sdk/core/azure-core-amqp/inc/azure/core/amqp/common/async_operation_queue.hpp

```diff
@@ -71,6 +71,13 @@ namespace Azure { namespace Core { namespace Amqp { namespace Common { namespace
       std::unique_lock<std::mutex> lock(m_operationComplete);
       m_operationQueue.clear();
     }
+    /** Returns the current number of elements in the queue.
+     */
+    size_t Depth()
+    {
+      std::unique_lock<std::mutex> lock(m_operationComplete);
+      return m_operationQueue.size();
+    }
```

Comment:
nit: rename to `Count()` or `Size()` instead, similar to  https://cplusplus.com/reference/queue/queue/size/